service/src/main/kotlin/fi/espoo/evaka/pedagogicaldocument/PedagogicalDocumentNotificationService.kt

```diff
@@ -83,11 +86,14 @@ class PedagogicalDocumentNotificationService(
                 }
                 .list()
         } catch (e: Exception) {
+            logger.error(e) { "Could not get pedagogical documentation notifications" }
```

Comment:
Rather than log the error should the `try ... catch` be removed altogether? Is it expected that the query fails?

Reply:
And if it's expected sometimes I would handle it explicitly and rethrow every other error.

Reply:
Failure was expected before this fix 😆 
But good point, removed it 👍